std_msgs/Header header
string objeto 
float64 topevirtual
